### PR TITLE
[OSDOCS-5527]: Adding 'Configuring hosted control planes' assembly

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.13+

Issue: https://issues.redhat.com/browse/OSDOCS-5527

Link to docs preview: https://57171--docspreview.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-configuring.html

QE review:
- [x] QE has approved this change.

Additional information: This PR adds an assembly to the "Hosted control planes" book that points to the getting started/configuring tasks for hosted control planes, which are published in the ACM docs.

### CODE REVIEW

**Comment by jiezhao16** [comment]
lgtm

File: hosted_control_planes/hcp-configuring.adoc [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+:FeatureName: Hosted control planes
+include::snippets/technology-preview.adoc[]
+
+You can view the procedures by selecting from one of the following providers:
+
+Amazon Web Services (AWS)
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/clusters/cluster_mce_overview#hosting-service-cluster-configure-aws[Configuring the hosted cluster on AWS]: Create the AWS S3 OIDC secret, create a routable public zone, enable external DNS, enable AWS PrivateLink, enable the hosted control planes feature, and install the hosted control planes CLI.
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/clusters/cluster_mce_overview#hosted-control-planes-manage-aws[Managing hosted control plane clusters on AWS]: Create, import, access, or delete a hosted cluster on AWS.
```
Comment (jeana-redhat) [inline hosted_control_planes/hcp-configuring.adoc:20]: The second two bullets feel more like complete sentences than the first two. I don't have a clever suggestion or anything but I wonder if they could all sound more similar :)

**Review by jeana-redhat — COMMENTED** [review]
super minor nits for your consideration ;)

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: hosted_control_planes/hcp-configuring.adoc [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+toc::[]
+
+To get started with hosted control planes for {product-title}, you first configure your hosted cluster on the provider that you want to use. Then, you complete a few management tasks. 
+
+:FeatureName: Hosted control planes
+include::snippets/technology-preview.adoc[]
+
+You can view the procedures by selecting from one of the following providers:
+
+Amazon Web Services (AWS)
```
[the remark this answers is not in the document]
Comment (lahinson) [reply hosted_control_planes/hcp-configuring.adoc:17]: I had wondered whether I should make them headings. I'll give it a try!

File: hosted_control_planes/hcp-configuring.adoc [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+* xref:../backup_and_restore/control_plane_backup_and_restore/disaster_recovery/dr-hosted-cluster-within-aws-region.adoc#dr-hosted-cluster-within-aws-region[Disaster recovery for a hosted cluster within an AWS region]: In a disaster situation, such as when a hosted cluster becomes read-only due to a failure, learn how to restore the hosted cluster.
+
+Bare metal
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/clusters/cluster_mce_overview#configuring-hosting-service-cluster-configure-bm[Configuring the hosted cluster on bare metal]: Configure DNS before you create a hosted cluster. 
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/clusters/cluster_mce_overview#hosted-control-planes-manage-bm[Managing hosted control plane clusters on bare metal]: Create a hosted cluster, create an InfraEnv, add agents, access the hosted cluster, scale the `NodePool` object, handle Ingress, enable node auto-scaling, or delete a hosted cluster.
```
[the remark this answers is not in the document]
Comment (lahinson) [reply hosted_control_planes/hcp-configuring.adoc:27]: It's a custom resource. Good call -- I think it should be in monospace. I'll fix that.

File: hosted_control_planes/hcp-configuring.adoc (hunk 1 above)
Comment (lahinson) [reply hosted_control_planes/hcp-configuring.adoc]: Good point. I updated those two sentences a little to (hopefully) make them sound more like the others.

**Pull request merged**